internal/gopsutil/common/common.go

```diff
@@ -366,7 +366,7 @@ func HostDev(combineWith ...string) string {
 // getSysctrlEnv sets LC_ALL=C in a list of env vars for use when running
 // sysctl commands (see DoSysctrl).
 func getSysctrlEnv(env []string) []string {
-	var foundLC bool
```

Comment:
@leonklingele Is here a way to just ignore all of `internal/gopsutil` directory via the new lint tool? Instead of having to add all these  `lint:ignore` directives?